sdk/samples/iot/iot_sample_common.c

```diff
@@ -33,14 +33,15 @@
 #include <openssl/evp.h>
 #include <openssl/hmac.h>
 
-#define IOT_SAMPLE_PRECONDITION_NOT_NULL(arg)   \
-do {                                            \
-    if (arg == NULL)                            \
-    {                                           \
+#define IOT_SAMPLE_PRECONDITION_NOT_NULL(arg) \
+  do \
+  { \
+    if (arg == NULL) \
+    { \

```

Comment:
Per C Guidelines, theses are supposed to be right justified. https://azure.github.io/azure-sdk/clang_design.html#clang-design-naming-macros3

Reply:
VS auto-formatted this. Is there a way to set `.clang-format` so that backslashes are right-justified?
Without that, enforcing such a guideline will become too cumbersome.

cc @antkmsft, @barcharcraz

Reply:
just discovered this too :)  Either clang-format needs to be updated or we change the guidelines.

Reply:
I talked with Anton and figured it out. We'll update `.clang-format` and run it across the files in the repo in a subsequent PR.

Good eye @momuno!

Reply:
Thanks @ahsonkhan for jumping on the fix so quickly!